mobile/src/bundles/users/components/new-account-header/step-track.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { Text, View } from '~/bundles/common/components/components';
+import { TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+type Properties = {
+    currentStep: number;
+    totalSteps: number;
+};
+
+const formatStep = (step: number, width: number): string => {
+    return String(step).padStart(width, '0');
+};
+
+const StepTrack: React.FC<Properties> = ({ currentStep, totalSteps }) => {
+    const stepWidth = 2;
```

Comment:
can be moved out of component, so it doesn't recreates on re-render 
```suggestion
    const stepWidth = 2;
```